lib/publish.ts

```diff
@@ -129,10 +152,12 @@ function doPublish(packageDir: string, gitRemoteUrl: string, params: Params): Pr
 
     function packPackageIntoTarball() {
         return directoryReady
-            .then(() => exec(`npm pack "${packageDir}"`, { cwd: packDir }))
+            .then(() => run([`npm`, `pack`, `"${packageDir}"`], { cwd: packDir }))
```

Comment:
The `"` make the command not work correctly

Reply:
I have removed the quotes and reworked a little bit for synchronous execution.